acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
`AssertAndStop(slice)` is inflexible since it can only do an exact match. I think I'd also prefer having tests that have particular event expectations read from the channel instead of constructing a slice to pass in to `AssertAndStop` (until `Node.Restart` and similar methods can be updated to record the correct expectations for use in `AssertAndStop`)